src/libs/kata-types/src/capabilities.rs

```diff
@@ -132,6 +132,6 @@ mod tests {
         // test append capabilities
         cap.add(CapabilityBits::GuestMemoryProbe);
         assert!(cap.is_mem_hotplug_probe_supported());
-        assert!(cap.is_fs_sharing_supported());
+        assert!(!cap.is_fs_sharing_supported());
```

Comment:
@Apokleos Why this change here?  On line 125 `FsSharingSupport` is being set and dont see it being unset anywhere. 
Why are you checking for false value here?

Reply:
Thx @amshinde good catch !
As L130 cap does set causing previous Caps replaced by `CapabilityBits::HybridVsockSupport` and then there's no other Caps existing. with such reason, the  `assert!(cap.is_fs_sharing_supported());` failed. 
I rechecked the test case, the root cause is L130. It should be changed to ` cap.add(CapabilityBits::HybridVsockSupport);` and the L135 keep it was.